pom.xml

```diff
@@ -16,18 +17,18 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <appengine.target.version>1.9.32</appengine.target.version>
+        <appengine.target.version>1.9.34</appengine.target.version>
```

Comment:
Also, did you sign the CAL?https://cla.developers.google.com/clas
That would be handy for future contributions. Thanks.